code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "bandaid"
 	item_state = "bandaid"
 	possible_transfer_amounts = null
-	volume = 50
+	volume = 40
```

Comment:
Stop doing this shit, and especially stop "forgetting" to mention it in the description.